### DISCUSSION TITLE
Chrome licenses stopped working?

### DISCUSSION THREAD

**Discussion by Zeeremixes** [aclap-dev/video-downloadhelper#51] [Help and Q&A] [answered]
### Your question

Hi!
From now I keep getting the same error message
Can you help me? As I don't have firefox licence for VideoDownloadHepler
The message is>

Warning on Chrome extensions and YouTube⊗
Unfortunately, the Chrome Web Store does not allow extensions for downloading YouTube videos 
so we could not include this feature in the Chrome version of the extension.
You can use Video DownloadHelper to download YouTube videos on the Firefox version.

version: 8.1.6.0
beta: false
build date: 2024-01-18
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only first 16 fisrt characters): 2c654168b5f44ce7
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\salsa",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

### Addon & CoApp version and details

version: 8.1.6.0
beta: false
build date: 2024-01-18
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only first 16 fisrt characters): 2c654168b5f44ce7
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\salsa",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment, marked as answer]
> Chrome licenses stopped working

It never worked (well… it kind of worked but many years ago).

As explained in the warning, Youtube is only downloadable if you're using Firefox or Edge.

**Reply by Zeeremixes** [reply]
Then ... Can I change my licence for Firefox Version? Without paying again

**Reply by paulrouget** [reply]
See the "License problem" section here: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem
